Doc/faq/programming.rst

```diff
@@ -293,7 +298,7 @@ It's good practice if you import modules in the following order:
 
 1. standard library modules -- e.g. ``sys``, ``os``, ``getopt``, ``re``
```

Comment:
```suggestion
1. standard library modules -- e.g. :mod:`sys`, :mod:`os`, :mod:`argparse`, :mod`re`
```

Use proper syntax, and update an old legacy module to a more modern and well-recognized alternative